internal/state/configuration.go

```diff
@@ -13,6 +14,9 @@ type Configuration struct {
 	// HTTPServers holds all HTTPServers.
 	// FIXME(pleshakov) We assume that all servers are HTTP and listen on port 80.
 	HTTPServers []HTTPServer
+	// HTTPSServers holds all HTTPSServers.
+	// FIXME(kate-osborn) We assume that all HTTPS servers listen on port 443.
+	HTTPSServers []HTTPServer
```

Comment:
The reason I keep them separate here is because I wanted to be able to add a default tls server block only if there were HTTPS servers defined. I tried implementing it with one slice of `HTTPServer` and with two, and the two-slice solution was much cleaner.

Reply:
kk. It looks a bit confusing though:

```go
// HTTPServers holds all HTTPServers.
HTTPServers []HTTPServer

// HTTPSServers holds all HTTPSServers.
HTTPSServers []HTTPServer
```

would it make sense to rename HTTPServer or have a separate type for HTTPSServer?

Reply:
I renamed the type `HTTPServer` to `VirtualServer`  and renamed the variable `HTTPSServers` to `SSLServers`. Let me know what you think.